tests-functional/clients/status_backend.py

```diff
@@ -154,6 +159,27 @@ def _set_proxy_credentials(self, data):
         data["StatusProxyStageName"] = "test"
         return data
 
+    def load_statusgo_data(self, path: str):
```

Comment:
```suggestion
    def extract_container_data(self, path: str):
```
or
```suggestion
    def extract_data(self, path: str):
```

Reply:
I didn't want to mention `container`, and made this function to be working with host as well, that's why there's a early return:
https://github.com/status-im/status-go/blob/d7f729889aec73d6ebdf8bed60817d690257d331/tests-functional/clients/status_backend.py#L163-L164

Let's do the `extract_data` 👍